client/src/Main.elm

```diff
@@ -2178,12 +2205,37 @@ updateCartQuantity (ProductId productId) quantity model =
                 Nothing ->
                     Just
                         { variant = Nothing
-                        , quantity = quantity
+                        , quantity =
+                            case param of
+                                Exact quantity ->
+                                    quantity
+
+                                Increase ->
+                                    defaultQuantity + 1
+
+                                Decrease ->
+                                    defaultQuantity
                         , requestStatus = RemoteData.NotAsked
                         }
 
                 Just v ->
-                    Just { v | quantity = quantity }
+                    Just
+                        { v
+                            | quantity =
+                                case param of
+                                    Exact quantity ->
+                                        quantity
+
+                                    Increase ->
+                                        v.quantity + 1
+
+                                    Decrease ->
+                                        if v.quantity == defaultQuantity then
```

Comment:
I might be missing something, but shouldn't we remove the item from the cart when the quantity is 0?

Reply:
Ah, I see this function doesn't actually update the cart, but rather updates the number in the product tile
<img width="304" height="451" alt="image" src="https://github.com/user-attachments/assets/29e63e5f-cc98-4a31-89e9-d19756a956e0" />

Apparently, `updateCartQuantity` name is a bit misleading then

Reply:
I've updated the name to `updateCartFormQuantity`, and added a comment for it. Hopefully, it is less confusing in the future.